assets/dev/js/frontend/modules.js

```diff
@@ -19,7 +18,7 @@ elementorModules.frontend = {
 		StretchedElement,
 		SwiperBase,
 		CarouselBase,
-		NestedTabs,
+		NestedTabs: () => import( /* webpackChunkName: 'nested-tabs-module' */ 'elementor/modules/nested-tabs/assets/js/frontend/handlers/nested-tabs' ),
```

Comment:
@rami-elementor 

I believe that this will reduce the file size of the `frontend-modules.js` file, when the mega-menu is not active on the page.

Are you able to check if this reduces the file size?

The next question that we check if it is safe to load the files like this, or that we would need to add a Promise for this.